.gitignore

```diff
@@ -11,7 +11,9 @@ phpunit.xml
 .env.local
 .env.*.local
 codeception.tric.yml
+codeception.slic.yml
```

Comment:
I know we used to ignore the `tric` configuration files in Common, but I think we should not: TEC and other plugins are including `tric` configuration files in VCS.